src/components/modules/InfoBox/index.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { PureComponent } from 'react';
+import { StyleSheet, View, Text, ImageBackground, Dimensions } from 'react-native';
+import { ModuleBox } from '@/components/modules';
+import { Image, Container, HtmlText } from '@/components/primitives';
+import { Responsive } from '@/components/utils';
+import { COLOR_MAP } from '@/constants';
+import { BREAKPOINTS } from '@/constants/index';
```

Comment:
these should be combined

Reply:
Combined imports for @constants. Could not do the same for @components.